docs/OURSPHINX-README.md

```diff
@@ -0,0 +1,10 @@
+# Details of Kytos Sphinx Theme 
+
+Here you'll find some directions and tips about the Kytos Sphinx Theme customization, it  all can be done right after cloning or forking this repository.  
+
+First of all, the locale Sphinx Theme must be the source for build the documentation. To do it, run `python setup.py develop` in `../kytos/sphinx-theme` path.
+
+## CSS STYLE
+To add some new CSS code, the file that should be edited is `kytos-sphinx.css_t`, and it can be found in this repository in `../kytos_sphinx_theme/kytos/static/` path. 
+
+Now the updates can be seen by using`make` in `../kytos/kytos/docs` path.
```

Comment:
I think the whole content of this PR can and should be on the main `/README.md`.

Reply:
I added into another folder because the /README.md seemed to be general instructions about installing sphinx theme, but I'll move the content!